oat-sa/extension-tao-outcomeui
# fix: leave columns building only in worker task runtime

This changes (first commit) fixing issue of failing CSV Result export UI due to a request running out of time.
Now we do not trigger `getColumnsToExport()` and do not instantiate `VariableColumn` classes and their data providers in scope of request which were produce significant load before.

How to test:
It is not possible to estimate impact through UI. But possible to check if no regression occurs.
- Make some deliveries passed by a test taker.
- Go to Results
- Check result table preview and export works fine with different options
- Check CSV results export for delivery or a class works fine
